src/psyclone/gocean1p0.py

```diff
@@ -3242,7 +3243,19 @@ def data_on_device(self, parent):
 
     def _read_from_device_routine(self, f2pygen_module=None, psyir=None):
         ''' Return the symbol of the routine that reads data from the OpenACC
-        device, if it doesn't exist create the Routine and the Symbol. '''
+        device, if it doesn't exist create the Routine and the Symbol.
```

Comment:
Sorrry to be a pain but on looking at this again I see that this doc string should also say that if either `f2pygen_module` or `psyir` are supplied then a suitable node representing the routine is inserted.

Reply:
You are right, I added the suggested docstring expansion.